components/script/dom/windowproxy.rs

```diff
@@ -388,46 +390,49 @@ impl WindowProxy {
         target: DOMString,
         features: DOMString,
     ) -> Option<DomRoot<WindowProxy>> {
-        // Step 3.
+        // Step 4.
         let non_empty_target = match target.as_ref() {
             "" => DOMString::from("_blank"),
             _ => target,
         };
-        // TODO Step 4, properly tokenize features.
         // Step 5
-        let noopener = features.contains("noopener");
-        // Step 6, 7
+        let features = tokenize_open_features(features);
```

Comment:
nit: I think it's much more clear to name the `Step 5` result as `tokenized_features` 👀?

```rust
// Step 5
let tokenized_features = tokenize_open_features(features);

// Step 7
let noopener = parse_open_feature_boolean(&tokenized_features, "noopener");
```